capnp-rpc-net/auth.mli

```diff
@@ -55,8 +55,9 @@ module Secret_key : sig
 
   val generate : unit -> t
   (** [generate ()] is a fresh secret key.
-      You must call [Nocrypto_entropy_lwt.initialize] before using this (it will give an
-      error if you forget). *)
+      You must call the relevant entropy initialization function
+    (e.g. {!Mirage_entropy_rng_unix.initialize}) before using this, or it
```

Comment:
`Mirage_crypto_rng_unix.initialize ()` it is these days :)

Reply:
Done